fineract-provider/src/main/resources/application.properties

```diff
@@ -165,6 +165,9 @@ fineract.content.s3.enabled=${FINERACT_CONTENT_S3_ENABLED:false}
 fineract.content.s3.bucketName=${FINERACT_CONTENT_S3_BUCKET_NAME:}
 fineract.content.s3.accessKey=${FINERACT_CONTENT_S3_ACCESS_KEY:}
 fineract.content.s3.secretKey=${FINERACT_CONTENT_S3_SECRET_KEY:}
+fineract.content.s3.region=${FINERACT_CONTENT_S3_REGION:us-east-1}
```

Comment:
why the default value? none of the other values has default value.

Reply:
> why the default value? none of the other values has default value.

@adamsaghy i followed whats in the current properties file. All AWS regions are defaulted to us-east-1 (cloudwatch and cloud)